StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/STPElementsSession.swift

```diff
@@ -31,6 +31,12 @@ final class STPElementsSession: NSObject {
     /// A map describing payment method types form specs.
     let paymentMethodSpecs: [[AnyHashable: Any]]?
 
+    /// An object containing Customer's saved payment methods information
+    let legacy_customer: STPLegacyElementsCustomer?
```

Comment:
We probably don't want to get rid of this class, because there will be a time where we eventually want to deserialize the default payment method.  Currently that is under discussion and we should not use whatever value is being sent to us, so we are not deserializing it.

Reply:
Ok, I don't feel strongly.  Can we take out "Legacy" from the name?  The ephemeral key is legacy but the information here is just about the non-legacy Customer object.  IMO I'd call it `STPElementsCustomerInformation`.